packages/twenty-shared/src/utils/strings/capitalize.ts

```diff
@@ -0,0 +1,7 @@
+import { isNonEmptyString } from '@sniptt/guards';
+
+export const capitalize = (stringToCapitalize: string) => {
```

Comment:
same as above:
In a PR right after I will replace all usages of isCompositeFieldMetadataType with this one. I just did not want to overload the PR